### ISSUE TITLE
Finalize / greatly-advance glossary term list and definition

### ISSUE THREAD

**Issue by jessecodescode** [caciviclab/odca-jekyll#291]
This issue came out of [this issue](https://github.com/caciviclab/odca-jekyll/issues/213).

Resources from previous ticket:

From @sfdoran:
Here is the CA secretary of state's glossary of CA campaign technical terms: https://www.sos.ca.gov/campaign-lobbying/cal-access-resources/cal-access-users-manual/glossary/

This is a CA Civic Lab glossary that I worked on in 2016. It still needs work, but it can get you started. https://docs.google.com/document/d/1tZpNzsKY_CQO4R52MoWAlGtbyufl_2eQ2yN8L3szg1Q/edit?usp=sharing

From @adborden: 
FYI there is also this glossary which is an older version than the
google doc
https://github.com/caciviclab/caciviclab.github.io/wiki/Glossary

**Comment by adborden** [comment]
@sfdoran do you want **all** the terms from the cal-access glossary?

**Comment by sfdoran** [comment]
No. I would start with terms useful for understanding the site that aren’t in the earlier glossary or terms that would benefit from have clearer definitions than those in the earlier glossary.
